src/App.js

```diff
@@ -10,16 +10,37 @@ import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-  // your config
-  apiKey: "AIzaSyAIk1t2yIMQaNi8AZM_IobGwey87LHSBcg",
-  authDomain: "fir-chat-edf7b.firebaseapp.com",
-  projectId: "fir-chat-edf7b",
-  storageBucket: "fir-chat-edf7b.appspot.com",
-  messagingSenderId: "502915838206",
-  appId: "1:502915838206:web:582f923f38ef38c39db5af",
-  measurementId: "G-34SHPKF8JS"
+  apiKey: "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  authDomain: "chat-app-15946.firebaseapp.com",
+  projectId: "chat-app-15946",
+  storageBucket: "chat-app-15946.appspot.com",
+  messagingSenderId: "533178884935",
+  appId: "1:533178884935:web:ef9483d776d0107ffe6262",
+  measurementId: "G-P58VBCQ0SY"
 })
 
+function Timebomb() {
```

Comment:
This function seems suspicious and not related, please review and let me know what you think.